WooCommerce/Classes/POS/Presentation/POSFloatingControlView.swift

```diff
@@ -21,8 +21,7 @@ struct POSFloatingControlView: View {
                     )
                 }
                 Button {
-                    presentationMode.wrappedValue.dismiss()
-                    // TODO: implement Get Support https://github.com/woocommerce/woocommerce-ios/issues/13401
+                    viewModel.showSupport = true
```

Comment:
Should we call a function here and change the boolean from the view model, rather than setting the property directly to `true`? This way we can test this behaviour, as well as change the access control of `showSupport` to `private(set)`

Reply:
When there's no logic to test (we're not responsible for testing whether we can set a Bool to true or false) then I don't think there's any value in that kind of extra complexity – it's just more code to maintain and the tests make it (slightly) harder to change, without providing a benefit.

It's more worthwhile if we do a little more in the button's action, like tracking an event.

Reply:
Yes, maybe it could help to test but only if we would write unit tests for `POSFloatingControlView`. It would likely mean mocking a view model and we could assert changes in a variable the same way we assert calls to a method.

For `PointOfSaleDashboardViewModelTests` it would allow writing tests to assert that `showSupportForm` sets `showSupport` to `true` and `hideSupportForm` sets `showSupport` to `false`. However, it doesn't add much benefit.

> as well as change the access control of showSupport to private(set)

Unfortunately, we couldn't change it to `private (set)` since `sheet` presentation requires both setter and getter to be visible. When the view is dismissed, `sheet` sets the value to `false`.